local-tests/test.ts

```diff
@@ -102,6 +102,8 @@ import { testFailImportWrappedKeysWithInvalidSessionSig } from './tests/testFail
 import { testFailImportWrappedKeysWithExpiredSessionSig } from './tests/testFailImportWrappedKeysWithExpiredSessionSig';
 import { testExportWrappedKey } from './tests/testExportWrappedKey';
 // import { testRemoveDecryptFailingOutsideLitAction } from './tests/testRemoveDecryptFailingOutsideLitAction';
+import { testSignMessageWithSolanaEncryptedKey } from './tests/testSignMessageWithSolanaEncryptedKey';
+import { testSignTransactionWithSolanaEncryptedKey } from './tests/testSignTransactionWithSolanaEncryptedKey';
```

Comment:
> Let's add another test for only signing and not broadcasting like signTransactionWithEncryptedKey

This is what the current `testSignTransactionWithSolanaEncryptedKey` is doing. It has `broadcast` set to `false` since sending a transaction would require testnet funds on Solana devnet

Reply:
Okay we aren't testing broadcasting in the tests? You just tested manually?

Reply:
@DashKash54 Correct. We'd need to extend the testing infra to generate a Solana key with test funds